### PR TITLE
Globally handle msrest ValidationError

### PR DESCRIPTION
**Pull request by tjprescott**
Adds msrest.exception.ValidationError to the global error handler so a stack trace is not thrown. These are not "unexpected" errors.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [N/A] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
                 "WaitOnNormalExit",
                 "RedirectOutput"
             ],
-            "debugStdLib": true
+            "justMyCode": false
```
Comment (marstr) [inline .vscode/launch.json]: I'd remove this kind of configuration change, as it isn't strictly part of solving the problem. :)

Comment (tjprescott) [reply .vscode/launch.json]: I could put it in a separate PR, but it is literally just because VSCode complains about it.

Comment (marstr) [reply .vscode/launch.json]: Fair enough, it's certainly not a big enough deal to block a PR about.

**Comment by tjprescott** [comment]
@adewaleo do you have those lines you want added for the ClientExceptionError?

File: src/azure-cli-core/azure/cli/core/util.py [hunk 2]
```diff
@@ -23,14 +23,17 @@
 def handle_exception(ex):
     # For error code, follow guidelines at https://docs.python.org/2/library/sys.html#sys.exit,
     from msrestazure.azure_exceptions import CloudError
-    from msrest.exceptions import HttpOperationError
+    from msrest.exceptions import HttpOperationError, ValidationError
```
Comment (adewaleo) [inline src/azure-cli-core/azure/cli/core/util.py]:
```suggestion
    from msrest.exceptions import HttpOperationError, ValidationError, ClientRequestError
```

File: src/azure-cli-core/azure/cli/core/util.py [hunk 3]
```diff
@@ -23,14 +23,17 @@
...
     from azure.cli.core.azlogging import CommandLoggerContext
 
     with CommandLoggerContext(logger):
 
         if isinstance(ex, (CLIError, CloudError)):
             logger.error(ex.args[0])
             return ex.args[1] if len(ex.args) >= 2 else 1
+        if isinstance(ex, ValidationError):
+            logger.error('validation error: %s', ex)
+            return 1
```
Comment (adewaleo) [inline src/azure-cli-core/azure/cli/core/util.py]:
```suggestion
            return 1
            
       if isinstance(ex, ClientRequestError):
            logger.error("The request failed:\n")
            logger.error(ex)
            return 1
```

Comment (adewaleo) [reply src/azure-cli-core/azure/cli/core/util.py]: Some suggestions to handle `HttpConnection/ClientRequestError`s

see the following issues:
#9089, #9196,

**Comment by tjprescott** [comment]
@adewaleo I committed your suggested changes. Thanks!

**Pull request merged**